javascript/express/reduce_fingerprint.yml

```diff
@@ -1,16 +1,12 @@
 patterns:
   - pattern: |
-      $<EXPRESS>.disable($<POWERED_BY>)
+      $<_>.disable($<POWERED_BY>)
     filters:
-      - variable: EXPRESS
-        detection: javascript_express_reduce_fingerprint_express_init
```

Comment:
Could we remove the auxiliary rule `javascript_express_reduce_fingerprint_express_init` below?  I don't think it's used anywhere now

Reply:
That's the most important one. It's used in the `required_detection` :)
We ensure `express` is initialized within the file so that opens up the rule a little bit but not too much.

Reply:
Ah, okay - that makes more sense! Thanks for explaining it 🙏